collectors/networkviewer.plugin/README.md

```diff
@@ -0,0 +1,85 @@
+# networkviewer.plugin
+
+This plugin uses eBPF to bring connection information from the kernel ring to user ring. 
+
+The default behavior of this plugin is to monitor all connections from the privates IPs defined inside the RFC 1918 
+(https://tools.ietf.org/html/rfc1918) to any destination, this default behavior can be changed with the configuration file
+`networkviewer.conf` .
+
+## Only Linux for while
+
+This plugin for while works only on Linux, but we are monitoring the development of eBPF on other Operate Systems. For 
+Linux there is also a limitation, the collector won't work for kernel verson smaller than `4.11.0` .
```

Comment:
'On Linux, the plugin requires kernel version 4.11.0 or newer.'

People tend to have an easier time understanding a positive assertion about something than a negative one, especially when dealing with things like version ranges.

Reply:
Thanks!